### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by core23** [comment]
The branch is out of sync with the stable one

**Comment by jordisala1991** [comment]
Can restart build once #1205 is merged

**Comment by greg0ire** [comment]
Restarting the build

File: .travis.yml [hunk 1]
```diff
@@ -41,27 +37,31 @@ matrix:
       env: TARGET=docs
     - php: '7.1'
       env: TARGET=lint
-    - php: '5.3'
+    - php: '7.0'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '5.6'
+    - php: '7.1'
+      env: SYMFONY=2.3.*
```
Comment (jordisala1991) [inline .travis.yml]: This is not correct for master

File: .travis.yml [hunk 2]
```diff
@@ -41,27 +37,31 @@ matrix:
...
+    - php: '7.1'
+      env: SYMFONY=2.7.*
```
Comment (jordisala1991) [inline .travis.yml]: This is also not correct

**Comment by jordisala1991** [comment]
I am not sure how this works but dev-kit configuration for media-bundle looks all wrong compared to admin-bundle for example.

File: .travis.yml (hunk 1 above)
Comment (greg0ire) [inline .travis.yml]: Indeed, it looks funny.

**Comment by greg0ire** [comment]
It should really be otherwise : https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml#L272

**Comment by jordisala1991** [comment]
3.x branch is missing symfony 2.3 and 2.7 😱

**Comment by greg0ire** [comment]
But this PR is quite old, and that code block was changed (by me 15 days ago). Let's close it and wait or ask for another deploy.

**Comment by greg0ire** [comment]
> https://github.com/sonata-project/dev-kit/pull/154

Looks like this was a mistake, see https://github.com/sonata-project/dev-kit/pull/154

**Comment by jordisala1991** [comment]
I will a PR on dev-kit

**Comment by greg0ire** [comment]
First do a test PR on media, just to be sure please.

**Pull request closed without merging**